OPCB/functions/tankspawner/spawn_tank_server.sqf

```diff
@@ -40,57 +39,53 @@ if (_isAttack == 1) then
 	
 	[_helicopter] remoteExec ["CHAB_fnc_tank_restriction",0,true];
 
-} else {if (_vehicle in _staticType) then 
-	
-	{
+} else {
+	if (_vehicle in _staticType) then {
 	_helicopter = _vehicle createVehicle ([9767.66,9978.72,0]);
 	_helicopter setdir 0;
 	
 	_helicopter addMPEventHandler ["MPKilled",
 	{
-		if (local (_this select 0)) then {
-			_current_helis = missionNamespace getVariable ["MaxStatic",1];
-			_current_helis = _current_helis -1;
-			missionNamespace setVariable ["MaxStatic",_current_helis,true];
+		if (isServer) then {
+			MaxStatic = MaxStatic - 1;
+			publicVariable "MaxStatic";
 		};
 	}];
 
-} else  
-{
-	_helicopter = _vehicle createVehicle ([9767.66,9978.72,0]);	
-	_helicopter setdir 0;
-	
-	private _cargoIndex = -1;
-	_vehicle = toUpper _vehicle;
-	{
-		if ((_x select 0) == _vehicle) exitWith {
-			_cargoIndex = _foreachIndex;
+	} else  {
+		_helicopter = _vehicle createVehicle ([9767.66,9978.72,0]);	
```

Comment:
Hardcoded coordinates! (I think Frost might have fixed this in his PR though)

Reply:
pfffff don't know what to do with this honestly.